### PR TITLE
Change to using scale-up calculation

### PR DESCRIPTION
**Pull request by sonyahanson**
Updated table, captions, and text according to #9

### CODE REVIEW

**Comment by sonyahanson** [comment]
This is done. Merge whenever.

File: manuscript/ms.tex [hunk 1]
```diff
@@ -268,7 +268,7 @@ \subsection{PDB mining results}
 \subsection{Small-scale kinase expression test in \emph{E. coli}}
 
 A panel containing the 96 kinase domain constructs selected through our semi-automated method, was tested for expression in \emph{E. coli}. 
-From this initial test, 68 kinase domains showed detectable expression (yield of more than 2 ng/$\mu$l eluate) (Table~\ref{expression_table}).
+From this initial test, 52 kinase domains showed reasonable expression (yield of more than 2 mg/L in the eluate) (Table~\ref{expression_table}).
```
Comment (jchodera) [inline manuscript/ms.tex]: You have "2 mg/L in the eluate".  Do you mean "2 mg/L culture"?  

It might be good to revert this change, or to include both numbers:
"yield of more than 2 ng/$\mu$L eluate, which corresponds to 2 mg/L culture".

File: manuscript/ms.tex [hunk 2]
```diff
@@ -128,7 +128,7 @@ \section{Introduction}
...
 Since bacterial expression is often complicated by the need to tailor expression and purification protocols individually for each protein expressed, we wondered whether a simple, uniform, automatable expression and purification protocol could be used to express a large number of human kinases to produce a convenient bacterial expression library to facilitate kinase research and selective inhibitor development.
 As a first step toward this goal, we developed a structural informatics pipeline to use available kinase structural data and associated metadata to select constructs from available human kinase libraries to clone into a standard set of vectors intended for phosphatase coexpression.
-Automated expression screening in Rosetta2 cells found that 68 human kinase domains express with yields greater than 2 $\mu$g/mL, which should be usable for biochemical, biophysical, screening, and structural biology studies.
+Automated expression screening in Rosetta2 cells found that 52 human kinase domains express with yields greater than 2 mg/L, which should be usable for biochemical, biophysical, screening, and structural biology studies.
```
Comment (jchodera) [inline manuscript/ms.tex]: 2 mg/L culture

File: manuscript/ms.tex [hunk 3]
```diff
@@ -368,7 +368,7 @@ \subsection{Small-scale kinase expression test in \emph{E. coli}}
 \begin{figure*}[tb]
    \includegraphics[width=\columnwidth]{figures/caliper_image.png}
   \caption{{\bf Synthetic gel image rendering of highest expressing kinases.}
-   Caliper GX II synthetic gel image rendering of kinases expressing > 200 $ng/ \mu l$ eluate from microfluidic capillary electrophoresis quantitation.
+   Caliper GX II synthetic gel image rendering of kinases expressing > 25 mg/L from microfluidic capillary electrophoresis quantitation.
```
Comment (jchodera) [inline manuscript/ms.tex]: 25 mg/L culture

**Comment by jchodera** [comment]
Pointed out a few things to clarify!

Might also be good to insert one line in the methods about how you compute expected scale-up yield from what was measured (ng/uL eluate).

**Comment by jchodera** [comment]
I'll then post this to bioRxiv after you merge!

**Comment by sonyahanson** [comment]
This sufficient?

**Comment by jchodera** [comment]
Good enough for now!  I'll post!
